sem_covid/entrypoints/ml_dags/enrich_pipeline_dag.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Date    : 19.07.2021 
+# Author  : Stratulat Ștefan
+# File    : enrich_pipeline_dag.py
+# Software: PyCharm
+from typing import Type
+
+from sem_covid.adapters.dag.abstract_dag_pipeline import DagPipeline
+from sem_covid.services.enrich_pipelines.base_enrich_pipeline import BaseEnrichPipeline, BasePrepareDatasetPipeline
+
+
+class EnrichPipelineDag(DagPipeline):
+    """
+        This class aims to combine the pipeline of data preparation
+         and enrichment pipeline.
+    """
+
+    def __init__(self, textual_columns: list,
+                 ds_es_index: str,
+                 features_store_name: str,
+                 prepare_pipeline: Type[BasePrepareDatasetPipeline] = BasePrepareDatasetPipeline,
```

Comment:
I would re-name `prepare_pipeline` into `dataset_preparation_pipeline` and `enrich_pipeline` into `dataset_enrichment_pipeline`.
 Or something along these lines.  `dataset_enrichment_executable` ...